geode-dunit/src/main/java/org/apache/geode/cache30/MultiVMRegionTestCase.java

```diff
@@ -2808,15 +2782,15 @@ public void testReplicate() throws Exception {
     final Object key2 = "KEY2";
 
     Object[] v = new Object[3000];
-    Arrays.fill(v, new Integer(0xCAFE));
+    Arrays.fill(v, 0xCAFE);
 
     final Object value2 = Arrays.asList(v);
     final Object key3 = "KEY3";
     final Object value3 = "VALUE3";
 
-    Host host = Host.getHost(0);
-    VM vm0 = host.getVM(0);
-    VM vm2 = host.getVM(2); // use VM on separate shared memory in case shared regions
+
+    VM vm0 = VM.getVM(0);
+    VM vm2 = VM.getVM(2); // use VM on separate shared memory in case shared regions
 
     SerializableRunnable create = new CacheSerializableRunnable("Create Mirrored Region") {
```

Comment:
Consider extracting the blocks that intellij has flagged as duplicated code to private methods.